app/schemas/de.tum.in.tumcampusapp.database.TcaDb/3.json

```diff
@@ -2,18 +2,12 @@
   "formatVersion": 1,
   "database": {
     "version": 3,
-    "identityHash": "be961deb5640fa65588a23f4da4551ef",
+    "identityHash": "0e484c1625d62d684221015cbbbc1da1",
     "entities": [
       {
         "tableName": "Cafeteria",
-        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`distance` REAL NOT NULL, `id` INTEGER NOT NULL, `name` TEXT NOT NULL, `address` TEXT NOT NULL, `latitude` REAL NOT NULL, `longitude` REAL NOT NULL, PRIMARY KEY(`id`))",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`id` INTEGER NOT NULL, `name` TEXT NOT NULL, `address` TEXT NOT NULL, `latitude` REAL NOT NULL, `longitude` REAL NOT NULL, PRIMARY KEY(`id`))",
```

Comment:
Naughty! :santa: 
Please don't change existing schemas

Reply:
Oops, I thought that by annotating it `@Ignore`, it would be disregarded by Room. Fixed it now.